library-and-framework-list.json

```
@@ -2298,5 +2298,39 @@
         "test_level": "fully-tested"
       }
     ]
+  },
+  {
+    "artifact": "org.fugerit.java:fj-doc-base",
+    "description": "Core modules of Fugerit Venus Doc framework, contains XML source and simple markdown doc handler.",
```

For consistency with the other entry:
```suggestion
    "description": "Core modules of Fugerit Venus Doc framework. Contains XML source and simple markdown doc handler.",
```